MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -3051,7 +3052,7 @@ public void addMassRepairOption(MassRepairOption mro) {
             massRepairOptions.remove(foundIdx + 1);
         }
 
-        massRepairOptions.sort(Comparator.comparingInt(MassRepairOption::getType));
+        massRepairOptions.sort((mro1, mro2) -> (((Comparable<Integer>) mro1.getType().ordinal()).compareTo(mro2.getType().ordinal())));
```

Comment:
"improving readability" lol

Reply:
I wish Java would compare enums like most reasonable languages.

Reply:
Double checking the code, I removed this and swapped the base logic to the stream api. Backend we don't care about ordering